web/client/libs/ajax.js

```diff
@@ -92,6 +93,10 @@ axios.interceptors.request.use(config => {
     var uri = config.url || '';
     var sameOrigin = !(uri.indexOf("http") === 0);
     var urlParts = !sameOrigin && uri.match(/([^:]*:)\/\/([^:]*:?[^@]*@)?([^:\/\?]*):?([^\/\?]*)/);
+    if (uri.indexOf("http") === 0) {
+        // we are only interested in http and https requests
```

Comment:
This is wrong, you are not checking the request type, but looking for absolute URL! also relative URL request must be intercepted.